assets/blocks/course-actions-block/course-actions/course-actions-edit.js

```diff
@@ -44,18 +108,34 @@ const CourseActionsEdit = ( { className, context: { postType } } ) => {
 		);
 	}
 
+	// Set class name for course status.
+	className = classnames( className, `is-status-${ courseStatus }` );
+
 	return (
-		<div className={ className }>
-			<InnerBlocks
-				allowedBlocks={ [
-					'sensei-lms/button-take-course',
-					'sensei-lms/button-continue-course',
-					'sensei-lms/button-view-results',
-				] }
-				template={ innerBlocksTemplate }
-				templateLock="all"
-			/>
-		</div>
+		<CourseStatusContext.Provider
+			value={ {
+				courseStatus,
```

Comment:
It would be a good practice to add this to a `useMemo` or something like that. Otherwise, it could trigger [unexpected re-renders](https://reactjs.org/docs/context.html#caveats).

Reply:
Good catch! f9ae5a83f